utils/cli/lib/inquirer.js

```diff
@@ -0,0 +1,52 @@
+const inquirer = require('inquirer');
+
+const actions = {
+  DEPLOY_ECOSYSTEM: 'Deploy Ecosystem',
+  DEPLOY_CURVE: 'Deploy Curve'
+};
+
+const curveTypes = {
+  STATIC: 'Static',
```

Comment:
I feel like we could think of a better name than "Static" . . maybe "Constant" or "Flat" ??  Thoughts?